staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go

```diff
@@ -232,6 +234,7 @@ func ValidateCustomResourceDefinitionUpdate(ctx context.Context, obj, oldObj *ap
 		preexistingExpressions:                   findPreexistingExpressions(&oldObj.Spec),
 		versionsWithUnchangedSchemas:             findVersionsWithUnchangedSchemas(obj, oldObj),
 		celEnvironmentSet:                        environment.MustBaseEnvSet(environment.DefaultCompatibilityVersion()),
+		allowInvalidCABundle:                     allowInvalidCABundle(oldObj.Spec.Conversion),
```

Comment:
here, pass the whole object in and return true if the oldObj does not yet have an Established=True condition

Reply:
fixed

Reply:
The ratcheting case already allows an invalid CA bundle to be updated to remain an invalid CA bundle. This case is only targeting changing a valid CA bundle -> invalid CA bundle pre-serving. Is there a use case for supporting this?

Reply:
> The ratcheting case already allows an invalid CA bundle to be updated to remain an invalid CA bundle. This case is only targeting changing a valid CA bundle -> invalid CA bundle pre-serving. Is there a use case for supporting this?

I suppose not.